lib/index.ts

```diff
@@ -94,6 +95,12 @@ export default class VM extends AsyncEventEmitter {
       this._common = new Common(chain, hardfork, supportedHardforks)
     }
 
+    // Set list of opcodes based on HF
+    this._opcodes = getOpcodesForHF(this._common.hardfork()!)
+    if (this._opcodes === undefined) {
+      throw new Error('hereee')
```

Comment:
Is there eventually a more expressive error message? 😄 😄 😄

Reply:
Oops haha :smile: :smile: that was there for debugging :sweat_smile: will remove now